### PR TITLE
Inverse ufw tasks to be connection cut's tolerant - ssh acces could b…

### PR DESCRIPTION
**Pull request by my-kel**
…e blocked

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -6,4 +6,4 @@ Main Developers
 Contributors
 ============
 
-* You are welcome
```
Comment (davidfischer-ch) [inline AUTHORS:9]: Please keep this welcome message after your name :-)

**Comment by davidfischer-ch** [comment]
Good catch! I am eager to merge!

**Comment by my-kel** [comment]
Done! You can merge now ;)

**Comment by davidfischer-ch** [comment]
Thanks for contributing!

**Pull request merged**